server/ops/docker/build-static-svgbob

```diff
@@ -1,4 +1,7 @@
 # build static executable binary
 FROM ekidd/rust-musl-builder:1.56.1
+COPY Cargo.toml .
+
+RUN SVGBOB_VERSION=`cat Cargo.toml | sed -nr '/svgbob_cli =/{s/svgbob_cli = "([0-9]+.[0-9]+.[0-9]+)"/\1/;p;}'` \
```

Comment:
Maybe we can simplify the command using grep?

```
cat Cargo.toml | grep "svgbob_cli =" | sed -r 's/.*"([^"]+)"/\1/'
```

I don't think it needs to check the value enclosed in double quotes.

What do you think?

Reply:
Works for me, have updated it